bokeh/command/subcommands/tests/__init__.py

```diff
@@ -1,16 +1,32 @@
 basic_scatter_script = """
 import numpy as np
 from bokeh.plotting import figure
+from bokeh.io import curdoc
 N = 5
 x = np.linspace(0, 4*np.pi, N)
 y = np.sin(x)
 p1 = figure()
 p1.scatter(x,y, color="#FF00FF")
+doc = curdoc()
+doc.add_root(Row(p1, p2))
 """
 
 basic_svg_scatter_script = """
 import numpy as np
 from bokeh.plotting import figure
+from bokeh.io import curdoc
+N = 5
+x = np.linspace(0, 4*np.pi, N)
+y = np.sin(x)
+p1 = figure(output_backend="svg")
+p1.scatter(x,y, color="#FF00FF")
+doc = curdoc()
+doc.add_root(p1)
+"""
+
+multi_svg_scatter_script = """
+import numpy as np
+from bokeh.plotting import figure
 from bokeh.layouts import Row
 from bokeh.io import curdoc
 N = 5
```

Comment:
What's the rationale for putting this stuff in `__init__`? is it used in multiple test files?